content/en/armory-enterprise/release-notes/rn-armory-agent/agent-service/agent-service-v-1-0-6.md

```diff
@@ -0,0 +1,9 @@
+---
+title: v1.0.6 Armory Agent Service (2022-02-02)
+toc_hide: true
+version: 01.00.06
+
+---
+
+### Fixes
+* Issue in which non namespaced resources had "default" instead of an empty one, this was preventing operations on them or related to those kinds such as deploying "CustomResources".
```

Comment:
```suggestion
## Fixes

* Fixed an issue where non-namespaced resources had `default` instead of an empty one. This prevented operations on them or related to those kinds, such as deploying `CustomResources`.
```